style.css

```diff
@@ -276,6 +275,11 @@ header {
   transition: background-color 400ms linear, color 400ms linear;
 }
 
+#bottomLine {
+  background-color: #ebecf0;
+  border: none;
+}
```

Comment:
As suggested by the previous reviewer, the right color to use for the horizontal line should be `#F7F7F9` and not `#ebecf0` as used here, please I'd suggest you implement it as your design is already close to the figma template 😄

Reply:
@usorfaitheloho In the figma design it is provided the line with #ebecf0 . Still do I have to change to #f7f7f9 .
![image](https://github.com/Porkalaikarthika/Portfolio/assets/130765186/bddded7b-1f58-492e-a512-f191d8395339)